src/Sylius/Bundle/AdminBundle/Resources/views/AdminUser/_media.html.twig

```diff
@@ -0,0 +1,7 @@
+{% form_theme form '@SyliusUi/Form/imagesTheme.html.twig' %}
+
+<div class="ui segment" id="add-avatar">
+    <h4 class="ui dividing header">{{ 'sylius.ui.add_avatar'|trans }}</h4>
```

Comment:
This section should not be called `Add avatar`, but just `Avatar`. We do not describe what we are doing in the section headers :D